view/changes.twig

```diff
@@ -28,7 +28,11 @@
       {% for day,concepts in changeList %}
```

Comment:
Seems like GitHub is preventing me from commenting the appropriate spot, but one should check `trans` strings for the relevant `sr-only` parts as well as the relevant strings on the sidebar tab that are conditionally affected by the introduced functionality.